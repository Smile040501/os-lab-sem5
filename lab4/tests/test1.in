# Test 1
# ./lab -txam

3 3
1,3,5,
7,9,11,
13,15,17,

4 4
10,12,14,16,
18,20,22,24,
26,28,30,32,
34,36,38,40,

4 1
2,
4,
6,
8,