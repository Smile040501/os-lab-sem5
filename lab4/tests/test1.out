4 1
280,
440,
600,
760,
